web_widget_date_format/static/src/js/formats.js

```diff
@@ -0,0 +1,26 @@
+// © 2016- Leonardo Donelli (donelli at monksoftware it)
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+odoo.define('web_widget_date_format.formats', function(require) {
+    'use strict';
+
+    var time = require('web.time');
+    var formats = require('web.formats');
+
+    var original_format_value = formats.format_value;
+    var new_format_value = function(value, descriptor, value_if_empty) {
+        // in list view columns, format is passed in the date_format attribute
+        if (descriptor.date_format) {
+            var v = moment(time.auto_str_to_date(value));
```

Comment:
Sorry if this is a stupid question: is the moment.js lib already included in standard odoo?